### ISSUE TITLE
Warning: You may need an appropriate loader to handle this file type.

### ISSUE THREAD

**Issue by sw-yx** [graphql/graphiql#617]
hi folks! I am getting 6 big scary yellow warnings all of this nature when trying to use the `graphiql` React component.. how do i silence this please?

```
WARNING in ./~/graphql-language-service-interface/dist/getOutline.js.flow
Module parse failed: ROUTE_TO_ABOVEMENTIONED_FILE Unexpected token (11:12)
You may need an appropriate loader to handle this file type.
|  */
|
| import type {
|   Outline,
|   TextToken,
 @ ./~/graphql-language-service-interface/dist ^.*$
 @ ./~/graphql-language-service-interface/dist/GraphQLLanguageService.js
 @ ./~/graphql-language-service-interface/dist/index.js
 @ ./~/codemirror-graphql/hint.js
 @ ./~/graphiql/dist/components/QueryEditor.js
 @ ./~/graphiql/dist/components/GraphiQL.js
 @ ./~/graphiql/dist/index.js
 @ ./client/components/graphql.js
 @ ./client/components/user-home.js
 @ ./client/components/index.js
 @ ./client/routes.js
 @ ./client/index.js
```

**Comment by aperkaz** [comment]
Hello @sw-yx ,
Any news on this? I am getting the same warning...

**Comment by levindixon** [comment]
As a temporary solution I ended up installing `ignore-loader` and adding the following to my webpack config:

```js
{ test: /\.flow$/, loader: 'ignore-loader' }
```

**Comment by sw-yx** [comment]
no news, sorry. i just kept going because i had some time pressure. thanks very much for the tip @levindixon

**Comment by darimpulso** [comment]
@levindixon, @aperkaz  ...  i don't know if you have still this issue, but you can install and include graphql-tag and use the loader in webpack

` {
        "test": /\.graphql$|\.flow$/,
        "exclude": /node_modules/,
        "use": [
          {
            "loader": 'graphql-tag/loader'
          }
        ]
 }`

**Comment by dandean** [comment]
This fix for this issue is to have a Webpack Loader configured to strip flow types:

```
npm install remove-flow-types-loader
```

```
      {
        test: /\.js/, // assuming the files are named .js.flow 
        enforce: 'pre',
        use: ['remove-flow-types-loader']
      },
```

**Comment by benmosher** [comment]
Here in the future, I adapted @levindixon's solution into 
```js
{ test: /\.flow$/, use: 'null-loader' }
```
after `yarn add --dev null-loader`-ing.

I don't totally understand why this doesn't destroy it, but I'm using this in a devteam-facing server anyway.

**Comment by cindhyandhini333** [comment]
> Here in the future, I adapted @levindixon's solution into
> 
> ```js
> { test: /\.flow$/, use: 'null-loader' }
> ```
> 
> after `yarn add --dev null-loader`-ing.
> 
> I don't totally understand why this doesn't destroy it, but I'm using this in a devteam-facing server anyway.

**Comment by barbalex** [comment]
This is very annoying as it completely spams the output when building my app - serching for meaningful output is made nearly impossible :-(

**Comment by barbalex** [comment]
Seems related to https://github.com/graphql/graphiql/issues/882

**Comment by Gautierrr** [comment]
Since https://github.com/graphql/graphiql/commit/36ed6695ce2b3f2608a0a679d12df64ed7a9b2bb, temporary solutions based on flow types doesn't work anymore...

Now we have warning on `.ts`, `.ts.map` and `.js.map` files. 😞 

New possible solution:

```js
{
  test : /\.(ts|ts.map|js.map)$/,
  include : [".node_modules/graphql-language-service-interface"],
  use : "ignore-loader"
}
```

**Comment by acao** [comment]
@Gautierrr @sw-yx  @aperkaz  fyi this should be resolved as of `0.17.0`, sorry forgot to deduplicate these many issues! all of them should be resolved including the typescript ones. you can follow the [webpack example](packages/examples/graphiql-webpack) make sure to have the suggested babel config though!

if you still are having issues with 0.17.0 please re open.

**Comment by rodoabad** [comment]
@acao can you update the link to your webpack example?
